api/src/db/migrations/2024.06.07T00.36.14.clean-user-group-names.ts

```diff
@@ -1,12 +1,16 @@
 import type { Migration } from "@/db/umzug"
-import { UserGroup } from "@/models"
 
-export const up: Migration = async () => {
-  await UserGroup.findEach(async (userGroup) => {
-    await userGroup.update({
-      name: userGroup.name.trim().replace(/\s+/g, " "),
-    })
-  })
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async (transaction) => {
+    const [rows]: any[] = await queryInterface.sequelize.query('SELECT name from user_groups', { transaction });
```

Comment:
You can set the typing via `.query<{ name: string }[]>` and dropping the `: any[]`, which causes a linter error, and hides the error below where you reference "row.id" which doesn't exist in the row object.

You are also missing the query type parameter, see https://sequelize.org/docs/v6/core-concepts/raw-queries/